.env

```diff
@@ -8,4 +8,7 @@ XCUBE_VERSION=0.13.0
 XCUBE_CCI_INSTALL_MODE=release
 XCUBE_CCI_VERSION=0.9.8
 
-INSTALL_MOOC=0
\ No newline at end of file
+INSTALL_MOOC=0
```

Comment:
Is this needed? If not, please remove.

Reply:
MOOC notebooks are not needed, but this switch is set to not install anyway.